deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -163,6 +163,10 @@ spec:
           - name: GFD_USE_NODE_FEATURE_API
             value: {{ .Values.nfd.enableNodeFeatureApi | quote }}
         {{- end }}
+        {{- if typeIs "string" .Values.imexNodesConfigFile }}   
```

Comment:
Should this also check for a non-empty value?

Reply:
suggestion taken

Reply:
Does that mean that you updated this? Is the change still pending?